code/style.css

```diff
@@ -1,4 +1,344 @@
+  padding: 30px 0px 30px 25px;
+}
+
+header img {
+  width: 180px;
+}
+
+nav {
+  display: none;
+  padding: 0 25px;
+}
+
+nav a {
+  color: #000;
+  text-decoration: none;
+  margin-left: 16px;
+}
+
+nav a:hover {
+  color: #f8e1aa;
+  font-size: 120%;
+}
+
+.hamburger {
+  display: flex;
+  width: 35px;
+  height: 30px;
+  flex-direction: column;
+  justify-content: space-between;
+  margin-right: 25px;
+}
+
+.hamburger div {
+  width: 35px;
+  height: 4px;
+  background: #000;
+  border-radius: 5px;
+}
+
+/*** BIG NEWS ***/
+
+.big-news  {
+  display:flex;
+  flex-direction: column;
+  width: 100%;
+  max-width: 1100px;
+  margin: 0 auto;
+}
+
+.big-news img {
+  width: 100%;
+  align-self: center;
+}
+
+.big-news div {
+  margin: 0 16px;
+  text-align: center;
+}
+
+.big-news p {
+  text-align: left;
+}
+
+/*** ARTICLE NEWS GRID ***/
+
+.wrapper {
+  margin: 16px;
+  padding-top: 25px;
+}
+
+.wrapper div {
+  width: 100%;
+  text-align: center;
+}
+
+.grid-articles {
+  max-width: 1100px;
+  margin: 0 auto;
+  display: grid;
+  grid-gap: 16px;
+  grid-template-columns: repeat(1, 1fr);
+}
+
+/*** POLAROID IMAGES ***/
+
+.item {
+  filter: grayscale(100%);
+}
+
+.item a {
+  color: #000;
+  text-decoration: none;
+}
+
+.polaroid {
+  background: white;
+  padding: 16px;
+  border: 1px solid white;
+  box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2), 0 6px 20px 0 rgba(0,0,0,0.19);
+}
+
+.polaroid img {
+  max-width: 100%;
+  height: auto;
+}
+
+.item:nth-of-type(3n+1) {
+  transform: scale(0.8,0.8) rotate(-5deg);
+  transition: all 0.35s;
+}
+
+.item:nth-of-type(3n+2) {
+  transform: scale(0.8,0.8) rotate(5deg);
+  transition: all 0.35s;
+}
+
+.item:nth-of-type(3n+3) {
+  transform: scale(0.8,0.8) rotate(-3deg);
+  transition: all 0.35s;
+}
+
+.item:nth-of-type(3n+4) {
+  transform: scale(0.8,0.8) rotate(6deg);
+  transition: all 0.35s;
+}
+
+.item:hover {
+  filter: none;
+  transform: scale(1,1) rotate(0deg) !important;
+  transition: all 0.35s;
+}
+
+/*** ARTICLE NEWS ***/
+
+.article-container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  max-width: 1100px;
+  margin: auto;
+  margin-top: 45px;
+  padding: 16px;
+}
+
+.article-container a {
+  background: #fff;
+  width: 100%;
+  text-decoration: none;
+  color: #000;
+  padding: 16px;
+  margin: 25px 0;
+  text-align: center;
+}
+
+.article-container a:hover {
+  animation-name: scale-shake;
+  animation-duration: 0.8s;
+  box-shadow: none;
+}
+
+.article-container a img {
+  width: 90%;
+  margin: 16px;
+}
+
+/*** FOOTER ***/
+
+footer {
+  display: flex;
+  justify-content: center;
+  margin-bottom: 10px;
+}
+
+.footer {
+  color: #000;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin: 10px;
+}
+
+.footer div {
+  margin-top: 14px;
+}
+
+.footer a {
+  color: #000;
+  margin: 20px;
+  margin-right: 20px;
+  margin-left: 20px; 
+  font-size: 30px;
+}
+
+.footer a:hover {
+  color:#f8e1aa;
+  font-size: 220%;
```

Comment:
I just want to be sure - are you aware that `100%` of `font-size` is not `30px` in this case? `%` in `font-size` goes back to default value set by browser. In Chrome for example it's `16px`, so your `220%` will be `35px` rather than `66px`